assets/prototype/domain/eventEditor/data/queries/datetimes/test/data.ts

```diff
@@ -0,0 +1,89 @@
+import { GraphQLError } from 'graphql';
+
+import '../../../../types';
+import { GET_DATETIMES } from '../';
+
+const eventId = 100;
+
+export const request = {
+	query: GET_DATETIMES,
+	variables: {
+		where: {
+			eventId,
+		},
+	},
+};
+
+export const setup = () => {
+	window.eeEditorEventData = { eventId };
+};
```

Comment:
We might need a test utility folder for this kind of stuff.

Reply:
Yes, that's there for time being.

Reply:
:man_facepalming: 
sorry
:smiley: